Phone/app/src/main/res/values/strings.xml

```diff
@@ -8,6 +8,8 @@
     <string name="client_secret">570f169fa60a4a91a7f5d493a251f41b</string>
     <string name="redirect_uri">http://localhost:8888/callback/</string>
     <string name="playlist_uri">https://api.spotify.com/v1/playlists/04wwOyhtie5aEUbvMdAOii?si=e2b67652ad034ce4/tracks</string>
+    <string name="userID">reimantas.tiuninas</string>
```

Comment:
Shouldn't this be after the "Strings related to login" comment below?

Reply:
Don't think so because this is associated with the playlist retrieval, not the login. But actually it is not used anymore, so I just removed it altogether :)